web/src/main.tsx

```diff
@@ -1,4 +1,4 @@
-import App from "@/components/App";
+import App from "./App";
```

Comment:
please use absolute imports: https://memsql.atlassian.net/wiki/spaces/MCDB/pages/1185873978/Code+Style+Best+Practices+Guide#Absolute-imports-%3E-Relative-imports

Reply:
Any advantage over this import

Because Absolute import has one issue. If you rename a file you need to go to each and every file to update the rename

with this if you rename it will auto reflect in all import and code change

Reply:
not sure I got it. If you rename a file you still need to change it even if using a relative import.
Please, refer to the code style: "No reason whatsoever, just consistency.". There are things for which we don't have a specific reason but we decide on one approach to keep consistency.